src/SelectField/Select.js

```diff
@@ -60,16 +60,6 @@ export class Select extends Component {
         }
     }
 
-    componentDidUpdate(prevProps) {
```

Comment:
Changed the implementation here to be more consistent with how the other fields work.